erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```
@@ -279,6 +279,29 @@ def set_indicator(self):
 			self.indicator_color = "green"
 			self.indicator_title = _("Paid")
 
+	def before_print(self, settings=None):
```

Ruff (0.13.1)

Unused method argument: `settings`

(ARG002)